aws/alb.go

```diff
@@ -331,11 +335,23 @@ func findManagedLoadBalancers(svc elbv2iface.ELBV2API, clusterID string) ([]*Loa
 	return loadBalancers, err
 }
 
+func getListeners(alb elbv2iface.ELBV2API, loadBalancerARN string) ([]*elbv2.Listener, error) {
+	// TODO: paged results
```

Comment:
I already did it for ACM certificates, it was just one loop and watch out for NextToken:
https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/feature/auto-select-acm-cert/aws/acm.go#L95

Do you think it does not make sense to do it in this PR?
I am fine if you just want to get it merged, but if it does not bother then I would be happy to have it in.